docs/components/Sql.md

```diff
@@ -0,0 +1,69 @@
+Overview
+========
+
+Nussknacker `Sql` enricher can connect to SQL databases with HikariCP JDBC connection pool.
```

Comment:
One crucial information is missing - user has to provide database driver. It should be placed in flink /lib folder, otherwise metaspace leak may occur (https://ci.apache.org/projects/flink/flink-docs-stable/docs/ops/debugging/debugging_classloading/#unloading-of-dynamically-loaded-classes-in-user-code - we can add link to this part of documentation)

Reply:
I've added info